src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx

```diff
@@ -312,7 +312,7 @@ const TimeEntryForm = props => {
         : foundTask.classification.toLowerCase();
```

Comment:
```suggestion
        : foundTask.category.toLowerCase();
```
Getting "TypeError: Cannot read properties of undefined (reading 'toLowerCase')". This appears to remove the error and possibly fix the issues others have been having.

Reply:
On it. changed this